packages/manager/modules/pci/src/projects/project/serving/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_serving_title": "Serving Engine",
+  "pci_projects_project_serving_description_label": "Description",
+  "pci_projects_project_serving_region_label": "Région",
+  "pci_projects_project_serving_add_label": "Créer un namespace",
```

Comment:
Isn't there a French equivalent ?

Reply:
Unfortunately no words describe better than this one.